### DISCUSSION TITLE
what releation bewteen release/8.0 and release/8.0-stage

### DISCUSSION THREAD

**Discussion by srxqds** [dotnet/runtime#93956] [General] [answered]
what releation bewteen release/8.0, release/8.0-stage and tags,
I found the commit history are not included each other.

**Comment by jkotas** [comment, marked as answer]
release/X.Y are the branches that produce the shipping binaries. 

We use release/X.Y-stage branches to accumulate the servicing changes before they get pushed into release/X.Y branch. It has to do with our release management process. You can ignore the release/X.Y-stage branches.
